File: 준영/핸드폰 번호 가리기.js
function solution(phone_number) {
  var answer = ''
  for (let i = 0; i < phone_number.length - 4; i++) {
    answer += '*'
  }
  for (let i = phone_number.length - 4; i < phone_number.length; i++) {
    answer += phone_number[i]
  }
  return answer
}

File: 태성/0623/핸드폰번호가리기.py

phone_number = "01033334444"

def solution(phone_number):
    answer = ''
    arr = phone_number[len(phone_number ) -4:len(phone_number)]

    pre_arr = ''.join(["*"]*(len(phone_number)-4))

    answer = pre_arr +arr
    return answer

print(solution(phone_number))



